client/components/deposit-status-chip/index.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import strings from './strings';
+
+const mappings: {
+	[ key: string ]: string;
+} = {
+	estimated: 'estimated',
+	pending: 'processing',
+	in_transit: 'sent',
+	paid: 'sent',
+	failed: 'failed',
+};
+
+type DepositStatusChip = {
+	status?: string;
+	isCompact?: boolean;
+};
+
+/**
+ * Renders a deposits status chip.
+ *
+ * Based off of the Chip component found components/chip.
+ *
+ * @param {AccountOverview.Overview} props Deposits overview
```

Comment:
Is this a copy-paste typo?

Shouldn't it be something like this?
```suggestion
 * @param {DepositStatusChip} props Status and isCompat flag.
```

Reply:
Thank you Shendy, yes a copy-paste fail!